storm-data-contracts/src/main/java/com/forter/contracts/BaseContractsBoltExecutor.java

```diff
@@ -61,68 +61,62 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
     }
 
     @Override
-    public void execute(Tuple inputTuple) {
-        boolean fail = false;
+    public void execute(Tuple inputTuple, BasicOutputCollector collector) {
         TOutput output = defaultOutput;
+
+        RuntimeException exception = null;
+
         final Object id = inputTuple.getValue(0);
         try {
             final Object data = inputTuple.getValue(1);
             ValidatedContract validatedInputContract = transformAndValidateInput(data);
 
             if (!validatedInputContract.isValid()) {
-                reportInputContractValidationFailed(validatedInputContract);
-                fail = true;
+                throw new ContractViolationException(validatedInputContract);
```

Comment:
make CVE a fast exception.

    @Override
    public Throwable fillInStackTrace()
    {
        return this;
    }